solutions/remotemonitoring/scripts/individual/ui-config.yaml

```diff
@@ -43,11 +43,11 @@ spec:
             configMapKeyRef:
               name: deployment-configmap
               key: iothubmanager.webservice.url
-        - name: PCS_BINGMAP_KEY
+        - name: PCS_AZUREMAPS_KEY
```

Comment:
I believe this will require some updates to our documentation, do we have an issue filed to track the updates?

Reply:
Documentation updated.